tests/unit_tests/test_artefacts.py

```diff
@@ -1,10 +1,96 @@
+'''Tests the artefacts file.
+'''
+
 from unittest import mock
 from unittest.mock import call
-
+from pathlib import Path
 import pytest
 
-from fab.artefacts import ArtefactStore, ArtefactsGetter, FilterBuildTrees
-from fab.constants import BUILD_TREES, CURRENT_PREBUILDS
+from fab.artefacts import (ArtefactSet, ArtefactStore, ArtefactsGetter,
+                           CollectionConcat, CollectionGetter,
+                           FilterBuildTrees, SuffixFilter)
+
+
+def test_artefact_store() -> None:
+    '''Tests the ArtefactStore class.'''
+    artefact_store = ArtefactStore()
+    assert len(artefact_store) == len(ArtefactSet)
+    assert isinstance(artefact_store, dict)
+    assert ArtefactSet.CURRENT_PREBUILDS in artefact_store
```

Comment:
Why is `CURRENT_PREBUILDS` singled out for special testing?

Reply:
Aeh ... I guess because I forgot to remove it when I figured out how to loop over all enums :) I've removed this one test (since in the lines below we would get an exception an ArtefactSet is not in the ArtefactStore).